docs/pages/integrating-the-design-system-into-your-project.md

```diff
@@ -2,48 +2,42 @@
 title: Integrating the Design System into your project
 layout: variation
 section: getting-started
-description: >
-
+description: >-
```

Comment:
That was a typo or something, but it does make me wonder: Does our search index these description fields? Would it be helpful to put something here?

Reply:
Actually, I don't even see how to edit this in Netlify CMS. Halp?

Reply:
Netlify does its own editing. I assumed that's what is happening here.

To answer your search question though, yes the [description field is included](https://github.com/cfpb/design-system/blob/master/docs/_includes/search-scripts.html#L29).

Reply:
Is it possible to fix it in the Netlify CMS interface? Or do I have to manually push a commit to the branch?

Reply:
I think you would have to edit it in the markdown file and push it up but that it would get edited by netlify the next time this page is edited. What is wrong with this edit it's doing? Is it rendering wrong?

Reply:
> Is it rendering wrong?

No, it's just invisible metadata.

> What is wrong with this edit it's doing?

It's just… pointless? Why take a blank description and add a hyphen to it?

> I think you would have to edit it in the markdown file and push it up but that it would get edited by netlify the next time this page is edited.

And if I wanted to enter a description with things to make it more easily searchable, are you saying that you think future edits to the page in Netlify would clobber it and revert it to just a hyphen?

Reply:
> And if I wanted to enter a description with things to make it more easily searchable, are you saying that you think future edits to the page in Netlify would clobber it and revert it to just a hyphen?

Potentially, yes. I'm not sure what it edits, but it does edit markdown to conform to a particular style.

Reply:
> Potentially, yes. I'm not sure what it edits, but it does edit markdown to conform to a particular style.

It's not part of the Markdown, though, it's in the YAML front matter.

Reply:
Yea sorry that's what I meant, the "source" of the markdown. See e.g. https://github.com/cfpb/design-system/pull/488/files netlify will remove line breaks and such.

Reply:
OK, looking at that link, I finally understand what's happening. The _entire contents_ of this page are in the `description` field, because it's not a normal page. And I guess the `>-` is some special formatting thing that Netlify uses. The changes do look fine on the test server, so I'll go ahead and merge this.